admission-webhook/rockcraft.yaml

```diff
@@ -28,24 +28,23 @@ parts:
       (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
 
   admission-webhook:
-    plugin: go
+    plugin: nil
     build-snaps:
-      - go/1.17/stable
+      - go/1.21/stable
     source: https://github.com/kubeflow/kubeflow.git
-    source-tag: v1.7-branch 
+    source-tag: v1.9.0
+    source-depth: 1
+    source-subdir: components/admission-webhook
     build-environment:
       - CGO_ENABLED: 0
       - GOOS: linux
     override-build: |
-      cd components/admission-webhook 
-      go build -o webhook -a .
-      cp webhook $CRAFT_PART_INSTALL/webhook
-    organize:
```

Comment:
Why did we remove the `organize` instruction here that move the binary?

Reply:
I can roll back it, just did to keep similar to other rocks